spec/datadog/core/utils/network_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'spec_helper'
+
+require 'datadog/core/utils/network'
+
+RSpec.describe Datadog::Core::Utils::Network do
+  describe '.raw_ip_from_request_headers' do
+    context 'with default IP headers' do
+      context 'iterates over the deafult headers (DEFAULT_IP_HEADERS_NAMES) in order' do
```

Comment:
Minor: Another typo :)

```suggestion
      context 'iterates over the default headers (DEFAULT_IP_HEADERS_NAMES) in order' do
```